styles/style.css

```diff
@@ -187,6 +187,32 @@ form textarea.missing {
     border-bottom: 2px solid #8f131f;
 }
 
+form a {
```

Comment:
It would be better to put this in a different class as most of the links in this form will not have this style.